packages/astro/src/index.ts

```diff
@@ -1,13 +1,13 @@
-import { resolve } from 'node:path'
+import { join, resolve } from 'node:path'
 import { fileURLToPath } from 'node:url'
 import type { AstroIntegration } from 'astro'
 import type { VitePluginConfig } from '@unocss/vite'
 import VitePlugin from '@unocss/vite'
 import type { UserConfigDefaults } from '@unocss/core'
 import type { Plugin, ResolvedConfig } from 'vite'
+import { RESOLVED_ID_RE } from '../../shared-integration/src/layers'
```

Comment:
Uhmm, shouldn't use shared package?

Reply:
> Uhmm, shouldn't use shared package?

I'm not sure, the same usage is found in the vite plugin

Reply:
It's ok to have this, they file will be inline and bundled with this package